lib/auth/auth_with_roles.go

```diff
@@ -4590,6 +4590,7 @@ func (a *ServerWithRoles) GetSnowflakeSessions(ctx context.Context) ([]types.Web
 
 // ListSAMLIdPSessions gets a paginated list of SAML IdP sessions.
 func (a *ServerWithRoles) ListSAMLIdPSessions(ctx context.Context, pageSize int, pageToken, user string) ([]types.WebSession, string, error) {
+	// Check if this a proxy service.
```

Comment:
What does this comment mean? The code doesn't appear relayed to proxy in any way.

Reply:
Ahhh, leftover comment. I'll remove.